### ISSUE TITLE
error in training command

### ISSUE THREAD

**Issue by abhishekaich27** [Muzammal-Naseer/CDA#5]
What is the last argument '--rl' in the training command?

**Comment by Muzammal-Naseer** [comment]
Thank you for your interest in our work.

--rl flag was supposed to select relativistic cross-entropy.  train.py only contains relativistic cross-entropy loss in the case of untargeted attack so there is no need of --rl. I have fixed it.

Please let me know if you face any other issue. We are going to update this repository with more experiments, pretrained models, etc..Stay tuned.
